generator/src/transform/NewTransform.hx

```diff
@@ -73,12 +73,47 @@ class NewTransform {
 		return mk(def, h.pos);
 	}
 
+	static function validateMath(tex:String, pos:Position)
+	{
+		// FIXME this completly ignores that the return is async ; )
```

Comment:
@lehonma this is async... any idea on how to deasync this (blocking this function, but not the event loop)

Well, I guess we can simply make the main generation passes async : (